docs/autogenerate/TestProject.adoc

```diff
@@ -0,0 +1,39 @@
+
+The goal of this step is to make sure everything is set up correctly.
+
+1. Create a new repository `hercules-ci-test` in the GitHub account you have configured.
```

Comment:
This can be confusing as I'd imagine most users will have an existing repository. So it should say this step is optional unless you don't have a project yet.

I feel we're making our users feel very dumb with such instructions.